client/my-sites/customer-home/cards/education/earn/index.jsx

```diff
@@ -1,24 +1,40 @@
 import { useTranslate } from 'i18n-calypso';
 import { connect } from 'react-redux';
 import earnCardPrompt from 'calypso/assets/images/customer-home/illustration--secondary-earn.svg';
+import { COURSE_SLUGS } from 'calypso/data/courses';
+import { useRouteModal } from 'calypso/lib/route-modal';
 import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 import EducationalContent from '../educational-content';
+import PaymentsFeaturesModal from './payments-features-modal';
 
 export const EDUCATION_EARN = 'home-education-earn';
 
 const EducationEarn = ( { siteSlug } ) => {
 	const translate = useTranslate();
+	const { isModalOpen, openModal, closeModal } = useRouteModal(
+		'coursePaymentsModal',
+		COURSE_SLUGS.PAYMENTS_FEATURES
+	);
 
 	return (
 		<EducationalContent
 			title={ translate( 'Make money from your website' ) }
 			description={ translate(
 				'Accept credit and debit card payments on your website for just about anything.'
 			) }
-			links={ [
+			modalLinks={ [
 				{
 					calypsoLink: true,
 					url: `/earn/${ siteSlug }`,
+					ModalComponent: PaymentsFeaturesModal,
+					modalComponentProps: {
+						isVisible: isModalOpen,
+						onClose: ( event ) => {
+							event.preventDefault();
+							closeModal();
+						},
+					},
+					onClick: () => openModal(),
```

Comment:
[Not important] We can optimize this code a bit by removing the anonymous function wrapper and just supplying the actual function.
```suggestion
					onClick: openModal,
```